src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,283 @@
+```
+# Pulling the chart to your local
+$ helm registry pull quay.io/giantswarm/liveness-chart --dest /tmp/
+
+# (CLOUD) Parse the chart template to Kubernetes resources
+$ helm template /tmp/liveness/giantswarm_liveness-chart_XXX/liveness-chart /tmp/liveness.yaml
+
+# (GIANTNETES) Parse the chart template to Kubernetes resources (use the ingress port given by support)
+$ helm template /tmp/liveness/giantswarm_liveness-chart_XXX/liveness-chart /tmp/liveness.yaml --set ingress.port=30010
+
+# Inject the sidecar in the deployment and submit the resources to Kubernetes API
+$ istioctl kube-inject -f /tmp/liveness.yaml | k apply -f -
+```
+
+It will launch a deployment and service listening in the port 80. At the same time, it will create a virtual service redirecting the traffic to the liveness service. Also, there is a gateway to wire the virtual service up with the ingress gateway.
+
+### AWS verification
+
+Get service hostname given by the cloud provider. Run
+```
+$ kubectl get svc istio-ingressgateway -n istio-system -o jsonpath="{.status.loadBalancer.ingress[0].hostname}"
+```
+And it will return the URL which the deployed app should reply. As we have set wildcard `*` in the hostname of the virtual service all `/healthz` traffic will be forwarded to the service.
+
+Executing this curl command should give you a `200` response.
+```
+curl http://<ELB_HOSTNAME>/healthz -I
+```
+
+### Azure verification
+
+Get the ingress controller IP given by the cloud provider. Run
+```
+$ kubectl get svc istio-ingressgateway -n istio-system -o jsonpath="{.status.loadBalancer.ingress[0].ip}"
+```
+And it will return the URL which the deployed app should reply to. Same as AWS, the wildcard `*` set as the hostname in the virtual service will send all traffic from `/healthz` path to the liveness service.
+
+Running this curl command should give you a `200` response.
+```
+curl http://<IP>/healthz -I
+```
+
+## On premise (Giantnetes)
+
+As probably you may know our [`giantnetes`](https://docs.giantswarm.io/basics/onprem-architecture/) tenant pods live inside the control plane pods. It means Giant Swarm creates a forwarding between the external ingress service to the ingress placed inside the tenant cluster. By default, the tenant clusters has a nginx ingress controller allocated out of the box. But, in case you want to use istio ingress controller you need to change the current ingress controller (remember it will affect all current traffic for that tenant cluster) or ask our team to allocate a new redirection from the parent endpoint to the Istio controller.
+
+
+Modifying the port for the ingress gateway
+```
+kubectl edit svc istio-ingressgateway -n istio-system
+----
+  - name: http2
+    nodePort: <PORT_PROVIDED_BY_GS_OR_DEFAULT_ONE>
+    port: 80
+    protocol: TCP
+    targetPort: 80
+  - name: https
+    nodePort: <PORT_PROVIDED_BY_GS_OR_DEFAULT_ONE>
+    port: 443
+    protocol: TCP
+    targetPort: 443
+----
```

Comment:
Since the manifest snippet here is not terminal output, I would move it into it's own code block and add a sentence of explanation in top.